src/xrpld/app/tx/detail/NFTokenUtils.cpp

```diff
@@ -1004,6 +1004,18 @@ tokenOfferCreatePreclaim(
         }
     }
 
+    if (view.rules().enabled(fixEnforceNFTokenTrustline) && !amount.native())
```

Comment:
shouldnt this be fixEnforceNFTokenTrustlineV2?

Reply:
Please refer to this [comment](https://github.com/XRPLF/rippled/pull/5297/files#r2116358575), we need to test it